ci/ansible.cfg

```diff
@@ -1,6 +1,5 @@
 [defaults]
-#callback_whitelist = custom_logger
 callbacks_enabled = custom_logger
-callback_plugins = /home/zuul/src/github.com/infrawatch/feature-verification-tests/callback_plugins
+callback_plugins = ../callback_plugins
 # additional paths to search for roles
-roles_path = /home/zuul/src/github.com/infrawatch/feature-verification-tests/roles:/usr/share/ansible/roles:/etc/ansible/roles:~/.ansible/roles:/home/zuul/src/github.com/openstack-k8s-operators/ci-framework/roles
+roles_path = ../roles:/usr/share/ansible/roles:/etc/ansible/roles:~/.ansible/roles:../../ci-framework/roles
```

Comment:
ansible.cfg is only used within the zuul job , we need to validate this role_path in the ci job.

Reply:
This ansible.cfg is being used in this github action as well. I also use it locally with this modification.

Reply:
ok emma I tried this update for the role_path here and its working so we can proceed but I think https://review.rdoproject.org/zuul/build/0f8c3a6fe26846fd972bedff1c2956b6/log/controller/ci-framework-data/logs/ci_script_002_run_fetch_nodes_facts_and.log